cabal.project

```diff
@@ -117,6 +117,7 @@ package ouroboros-consensus-cardano-test
 
 package ouroboros-network
   tests: False
+  ghc-options: -XFlexibleContexts
```

Comment:
@newhoggy I think this needs an explanation. I did see the warning; but are we sure that this is the _right_ fix for ouroboros-network? And does this impact 8.10.7 negatively?

Reply:
This is just temporary to unblock finding all the other compile errors.  The fixes should eventually go into the repository itself.